pkg/download/abs.go

```diff
@@ -14,55 +14,30 @@ import (
 
 const (
 	MaxConcurrentFetch = 20
-	_urlPackagePath    = "%s/raw/packages/%s/trunk/PKGBUILD"
+	_urlPackagePath    = "https://gitlab.archlinux.org/archlinux/packaging/packages/0ad/-/raw/main/PKGBUILD"
```

Comment:
isn't `_urlPackagePath` unused?

Reply:
you're right, curious it wasn't flagged by the linter. Opening a PR now

Reply:
#2179